Sources/MapboxCoreNavigation/NativeHandlersFactory.swift

```diff
@@ -99,8 +108,26 @@ class NativeHandlersFactory {
                     endpointConfig: endpointConfig)
     }()
     
+    lazy var navigatorConfig: NavigatorConfig = {
+        return NavigatorConfig(voiceInstructionThreshold: nil,
+                               electronicHorizonOptions: nil,
+                               polling: nil,
+                               incidentsOptions: nil,
+                               noSignalSimulationEnabled: nil,
+                               avoidManeuverSeconds: NSNumber(value: RerouteController.DefaultManeuverAvoidanceRadius),
+                               useSensors: false)
+    }()
+    
     lazy var configHandle: ConfigHandle = {
-        let customConfig = UserDefaults.standard.dictionary(forKey: customConfigKey) ?? [:]        
+        let historyAutorecordingConfig = [
```

Comment:
It's not about auto-recording anymore. What if we call it `defaultConfig` or something like that?